src/views/Governance.vue

```diff
@@ -129,6 +127,14 @@ export default defineComponent({
       releaseLoading()
     }
 
+    const getProposalStatistic = async () => {
+      const res = await callers.getProposalsStatistic()
+      const proposalsStatistic = await res.json()
```

Comment:
await res.json() ?

Try use destructurization, for example
const { data } = await callers.getProposalsStatistic()